### PR TITLE
Updated and removed links

### PR DESCRIPTION
**Pull request by slayeh17**
removed [https://layer5.io/community/members/navendu-pottekkat](url)
changed [https://layer5.io/community/members/anita-charles](url) to [https://layer5.io/community/members/anita-ihuman](url)

Signed-off-by: Debak Roy <104914725+slayeh17@users.noreply.github.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 526b487532ef112492a3008085471082231f86d8 at: https://62f5e022b1a34c6ecc2bdf0b--layer5.netlify.app

File: src/collections/programs/sca-contributhon/index.mdx [hunk 1]
```diff
@@ -180,7 +180,6 @@ May 8, 2021 to May 14, 2021 - SCA announces successful participants and awards $
 <br />
 <b>Mentors</b>
```
Comment (Nikhil-Ladha) [inline src/collections/programs/sca-contributhon/index.mdx:181]: Remove this section completely

Comment (Nikhil-Ladha) [reply src/collections/programs/sca-contributhon/index.mdx:181]: Mentors section

Comment (slayeh17) [reply src/collections/programs/sca-contributhon/index.mdx:181]: Ok thank you for telling.

**Comment by slayeh17** [comment]
@Nikhil-Ladha can you check please I am new to open source actually so I can't really say if the updation took place or not.
Thank you.

File: src/collections/programs/sca-contributhon/index.mdx (hunk 1 above)
Comment (Nikhil-Ladha) [reply src/collections/programs/sca-contributhon/index.mdx:176]: You removed the mentors section at the wrong line.
You need to remove it here at line: 176

**Comment by l5io** [comment]
🚀 Preview for commit 2c732e2a5d770c50ce64fd645c1c755299967688 at: https://62f5ee53850fce75bc734412--layer5.netlify.app

File: src/collections/programs/sca-contributhon/index.mdx (hunk 1 above)
Comment (slayeh17) [reply src/collections/programs/sca-contributhon/index.mdx:176]: Oh I am sorry, I will close the PR and try again.
Thank you.

**Pull request closed without merging**